lib/bloc/podcast/podcast_bloc.dart

```diff
@@ -436,13 +444,26 @@ class PodcastBloc extends Bloc {
   }
 
   void applySearchFilter() {
```

Comment:
The method here is labelled `applySearchfilter` and handles the search feature on the podcast details page. By deleting the `searchTerm` handling the search feature is now broken. If you give it a try you'll see you can no longer search episodes.